components/Underwaterb.vue

```diff
@@ -975,8 +1011,95 @@ class Underwater {
 
         resolve();
       };
-    });
-    return p;
+    }).then((mainMesh) => {
+      const fileToFetch = '/bakedAnim/tartaruga.json';
+      console.log('Fetching Turtle: ' + fileToFetch);
+      return fetch(fileToFetch);
+    }).then((response) => {
+      if (!response.ok) {
+        // Constroi o Baking porque não existe
+        const baker = new BABYLON.VertexAnimationBaker(this.scene, mainMesh);
+        baker.bakeVertexData(animationRanges).then((vertexData) => {
+          console.log('Serializando Baking');
+          const vertexDataJSON = baker.serializeBakedVertexDataToJSON(vertexData);
+          const a = document.createElement('a');
+
+          a.setAttribute('href', 'data:text/plain;charset=utf-8,' + encodeURIComponent(vertexDataJSON));
+          a.setAttribute('download', 'tartaruga.json');
+          a.click();
+
+          const vertexTexture = baker.textureFromBakedVertexData(vertexData);
+          const bakedVertexAnimationManager = new BABYLON.BakedVertexAnimationManager(this.scene);
+          bakedVertexAnimationManager.texture = vertexTexture;
+
+          bakedVertexAnimationManager.setAnimationParameters(
+            animationRanges[0].from,
+            animationRanges[0].to,
+            0,
+            30
+          );
+
+          const anim = new BABYLON.Vector4(
+            animationRanges[0].from,
+            animationRanges[0].to,
+            0,
+            30
+          );
+
+          mainMesh.bakedVertexAnimationManager = bakedVertexAnimationManager;
+          //        this.scene.stopAnimation(mainMesh);
+
+          // set animation parameters
+
+          animParameters.set(anim.asArray(), 0);
+          mainMesh.thinInstanceSetBuffer('bakedVertexAnimationSettingsInstanced', animParameters, 4);
+          // container.animationGroups.map(g => g.pause());
+          //          this.scene.registerBeforeRender(() => {
+          //            mainMesh.bakedVertexAnimationManager.time += this.engine.getDeltaTime() / 1000.0;
+          //          });
+        });
+        throw new Error('Error loading JSon');
+      }
+      return response.text();
```

Comment:
deveria ser `response.json()` se é um json.

Reply:
Então, ele salva o JSon, mas o método de loading to babylon espera um text... Como não faz nada com o json depois dessa parte, eu deixei passar como text mesmo.